columnflow/tasks/histograms.py

```diff
@@ -185,7 +185,7 @@ def run(self):
                 fill_kwargs = {
                     "category": events.category_ids,
                     "process": events.process_id,
-                    "shift": self.global_shift_inst.id,
+                    "shift": ak.Array(np.zeros(len(events)) + self.global_shift_inst.id),
```

Comment:
Is the `ak.Array` needed here? If not, then what is typically done is

```python
np.ones(len(events), dtype=np.int32) * self.global_shift_inst.id
```

Esp. the dtype could be needed here as `zeros` could default to `float64`.

Reply:
changed :+1: